mesos/task.go

```diff
@@ -45,6 +48,34 @@ func createTaskInfo(task eremetic.Task, offer *mesosproto.Offer) (eremetic.Task,
 	return task, taskInfo
 }
 
+func buildDockerCliParameters(task eremetic.Task) []*mesosproto.Parameter {
+	//To be able to move away from docker CLI in future, parameters aren't fully exposed to the API
+	params := make(map[string]string)
+	if task.DNS != "" {
+		params["dns"] = task.DNS
+	}
+	var parameters []*mesosproto.Parameter
+	for k, v := range params {
+		parameters = append(parameters, &mesosproto.Parameter{
+			Key: proto.String(k),
+			Value: proto.String(v),
+		})
+	}
+	return parameters
+}
+
+func buildNetwork(task eremetic.Task) *mesosproto.ContainerInfo_DockerInfo_Network {
```

Comment:
This looks a very round about way of doing it. I think this (untested) should do the same

```
func buildNetwork(task eremetic.Task) *mesosproto.ContainerInfo_DockerInfo_Network {
    if task.Network == "" {
        return mesosproto.ContainerInfo_DockerInfo_BRIDGE.Enum()
    }
    return mesosproto.ContainerInfo_DockerInfo_Network(mesosproto.ContainerInfo_DockerInfo_Network_value[task.Network]).Enum()
}
```